cmds/exp/pxeserver/main.go

```diff
@@ -90,14 +95,18 @@ func (s *server) dhcpHandler(conn net.PacketConn, peer net.Addr, m *dhcpv4.DHCPv
 		log.Printf("Could not create reply for %v: %v", m, err)
 		return
 	}
-	if _, err := conn.WriteTo(reply.ToBytes(), &net.UDPAddr{IP: net.IP{255, 255, 255, 255}, Port: 68}); err != nil {
+	v("Sending %v to %v", reply.ToBytes(), s.bcast)
```

Comment:
Just log.Printf it. And use reply.Summaty() instead of ToBytes or reply.String if you want a short version

Reply:
Whoa, that output is amazing! Done.